docs/source/how-to-upstream.mdx

```diff
@@ -39,24 +39,31 @@ Specify the path of the file to upload, where you want to upload the file to in
 ```py
 >>> from huggingface_hub import HfApi
 >>> api = HfApi()
->>> api.upload_file(path_or_fileobj="/path/to/local/folder/README.md", 
-...                 path_in_repo="README.md", 
-...                 repo_id="username/test-dataset",
-...                 repo_type="dataset",
+>>> api.upload_file(
+...   path_or_fileobj="/path/to/local/folder/README.md",
+...   path_in_repo="README.md",
+...   repo_id="username/test-dataset",
+...   repo_type="dataset",
 ... )
```

Comment:
Nitpick, we use 4-space formatting across the repo

Reply:
Oops yep of course 👍